recurring_ical_events.py

```diff
@@ -119,12 +119,21 @@ class Repetition:
     ]
 
     def __init__(self, source, start, stop, keep_recurrence_attributes=False):
```

Comment:
Sorry for jumping in too late here. It would be nice to use python type annotations here, so we know in which format start and stop should be and under which conditions they are inclusive and when not.

Reply:
Yep. I think, when icalendar releases a version 5, we can also drop Python < 3.7 #94.

Reply:
Which Python version do you need for annotations?

Reply:
It was introduced in 3.5